index.js

```diff
@@ -0,0 +1,40 @@
+const openModalButtons = document.querySelectorAll('[data-modal-target]')
+const closeModalButtons = document.querySelectorAll('[login-popup-close-modal]')
```

Comment:
Let's make `js` or `src` folder and create three files:
1. openLoginPopup.js -> default export handleOpenLoginPopup
2. closeLoginPopup.js -> default export handleCloseLoginPopup
3. followButton.js -> default export handleFollowButtonClick

Import all 3 in index.js and call them